docs/datasets/index.md

```diff
@@ -0,0 +1,78 @@
+# Datasets
+
+Autoware partners provide datasets for testing and development. These datasets are available for download here.
+
+## Bus-ODD (Operational Design Domain) datasets
+
+### Leo Drive - ISUZU sensor data
+
+This dataset contains data from the Isuzu bus used in the Bus ODD project.
+
+The data contains data from following sensors:
+
+- 1 x VLP16
+- 2 x VLP32C
+- 1 x Applanix POS LV 120 GNSS/INS
+- 3 x Lucid Vision Triton 5.4MP cameras (left, right, front)
+- Vehicle status report
+
+It also contains /tf topic for static transformations between sensors.
+
+#### Required message types
+
+The GNSS data is available in `sensor_msgs/msg/NavSatFix` message type.
+
+But also the Applanix raw messages are also included in `applanix_msgs/msg/NavigationPerformanceGsof50` and `applanix_msgs/msg/NavigationSolutionGsof49` message types.
+In order to be able to play back these messages, you need to build and source the `applanix_msgs` package.
+
+```bash
+# Create a workspace and clone the repository
+$ mkdir -p ~/applanix_ws/src && cd "$_"
+$ git clone https://github.com/autowarefoundation/applanix.git
+$ cd ..
+
+# Build the workspace
+$ colcon build --symlink-install --packages-select applanix_msgs
+
+# Source the workspace
+$ source ~/applanix_ws/install/setup.bash
+
+# Now you can play back the messages
+```
+
+Also make sure to source Autoware Universe workspace too.
+
+#### Download instructions
+
+```bash
```

Comment:
When you have console output, using `console` is better.

``````suggestion
```console
``````

Reply:
I didn't know this, thanks.